geometry.go

```diff
@@ -1,5 +1,12 @@
 package fyne
 
+// Vec2 marks geometry types that can operate as a coordinate vector
```

Comment:
https://github.com/fyne-io/fyne/wiki/Proposal:-Use-Vectors-to-Represent-Position,-Size,-Etc.

I will see if the other name suggestions work better

Reply:
I don't think Coordinate is better than Vector. It is generic on purpose - it can refer to a coordinate, a size, a delta etc. It's just X and Y quantity of some sort, and I can't find a better name for it

Reply:
Of course we could make `Vector` a more specific `XYDelta` (as that is what what it's been used for so far), and then rename `Vec2` as `Vector` or `Vector2`.
The rest of the naming is now tidied up.

Reply:
Okay. I like the idea of using `Vector` for the interface.
The mathematical correct name for the current `Vector` implementation would be `Displacement` (https://en.wikipedia.org/wiki/Displacement_(geometry)). Unless you or someone else has a better idea, I would favour this one.

Reply:
> The mathematical correct name for the current `Vector` implementation would be `Displacement`

Good point, though not a commonly used phrase and has a very specific meaning whereas this is trying to be quite generic. Delta feels more commonly used, or we could even go simpler with `Move` - this will be alongside `Size` and `Position` so looking for something commonplace word.

Reply:
I’m okay with `Move` or `Movement` but don’t really like `Delta` :).